src/connectedmachine/azext_connectedmachine/generated/_help.py

```diff
@@ -0,0 +1,127 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['connectedmachine machine'] = """
+    type: group
+    short-summary: connectedmachine machine
+"""
+
+helps['connectedmachine machine list'] = """
+    type: command
+    short-summary: "Lists all the hybrid machines in the specified subscription. Use the nextLink property in the \
+response to get the next page of hybrid machines."
```

Comment:
The nextLink property is not exposed in CLI command, so the second sentence can be removed in the help message.

Reply:
Addressed in https://github.com/Azure/azure-rest-api-specs/pull/10795 , please review.

Reply:
I manually fixed this in the last commit